debian.debian/changelog

```diff
@@ -0,0 +1,8 @@
+rust-luwen (0.0.1) unstable; urgency=low
+  [ Evangelos Ribeiro Tzaras ]
```

Comment:
Nit: Who is this?

Reply:
that might be copy/paste, easy enough to pull/change quick

Reply:
sorry that comes out of the git build-package stuff, we can swap it out if you want it's mostly a placeholder